doc/nn_geterror.txt

```diff
@@ -0,0 +1,51 @@
+nn_geterror(3)
+===========
+
+NAME
+----
+nn_geterror - retrieve the current error for an endpoint
+
+
+SYNOPSIS
+--------
+*#include <nanomsg/nn.h>*
+
+*int nn_geterror (int s, int ep);*
+
+
+DESCRIPTION
+-----------
+It allows at any time request for an endpoint the last error set due to a condition 
+that prevented the endpoint operating normally.
```

Comment:
English wordsmithing (I guess English isn't your first language.)  Try this:

Retrieves the last error condition for a given endpoint on the socket.  The returned value is the same as for 'nn_errno(3)' except that the error scope is limited to a specific endpoint.